hapi19/routes/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.register = (server, options) => {
+  server.route([
+    {
+      method: 'GET',
+      path: '/',
+      handler: {
+        view: 'index'
+      }
+    },
+    {
+      method: 'GET',
+      path: '/public/app.css',
+      handler(request, h) {
+        return h.file('public/app.css');
```

Comment:
I fixed this in master so you apps work when starting in root of NodeTestBenches

```suggestion
        return h.file(`${__dirname}/public/app.css`);
```